assets/src/domain/eventEditor/ui/ticketAssignmentsManager/data/useOnSubmitAssignments.ts

```diff
@@ -42,16 +57,30 @@ const useOnSubmitAssignments = () => {
 			if (ticketsToUpdate.length < datesToUpdate.length) {
 				ticketsToUpdate.forEach(([id, possibleRelation]) => {
 					const datetimes = possibleRelation?.datetimes || [];
-					updateTicket({ id, datetimes });
+					const input: UpdateTicketInput = { id, datetimes };
+					// if an entry exists in changed quantity map
+					// lets use this oppurtunity to update the quantity here
+					// to reduce the number of mutation requests
+					if (ticketsWithChangedQuantity?.[id]) {
+						input.quantity = ticketsWithChangedQuantity?.[id];
```

Comment:
this is used twice, so let's give it a name, so we don't have to check twice which is what

Reply:
Done